bot/exts/utils/makegurkan.py

```diff
@@ -0,0 +1,27 @@
+from discord.ext import commands
+from discord.utils import find
+import re
+
+
+class MakeGurkan(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.Cog.listener()
+    async def on_member_update(self, before, after):
```

Comment:
```suggestion
    async def on_member_update(self, before: discord.Member, after: discord.Member) -> None:
        """When the member updates their nicknames, this function will check if member is classified to be a gurkan or not."""
```
Remember to import `discord`